modules/shared/src/main/scala/org/tessellation/tools/DAGBlockGenerator.scala

```diff
@@ -24,25 +23,25 @@ import fs2.Stream
 object DAGBlockGenerator {
```

Comment:
I guess it's just `BlockGenerator` now

Reply:
I'd like to keep it DAGBlockGenerator, since we use it for DAG only and I'm not sure it can be 1:1 used somewhere else. Also, there are methods like createDAGs, etc. I'd have to refactor this generator which I'd like not in this PR.

Reply:
Noted to take care of it in next PRs